### PR TITLE
Changes to code style

### PR DESCRIPTION
**Pull request by cserb**
* followed ameba linting https://crystal-ameba.github.io/
* followed crystal coding style convention https://crystal-lang.org/reference/conventions/coding_style.html
* removed unreachable code

### CODE REVIEW

**Comment by cserb** [comment]
~~Forgot to bump up the version number, as it’s not backwards compatible due to renaming EC_Point and ECDSA_Signature~~

File: docs/Secp256k1/ECDSA_Signature.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 </svg>
 
   <meta id="repository-name" content="github.com/q9f/secp256k1.cr">
-  <title>Secp256k1::ECDSA_Signature - github.com/q9f/secp256k1.cr</title>
```
Comment (q9f) [inline docs/Secp256k1/ECDSA_Signature.html:18]: this file should be removed

File: docs/Secp256k1/EC_Point.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
 </svg>
 
   <meta id="repository-name" content="github.com/q9f/secp256k1.cr">
-  <title>Secp256k1::EC_Point - github.com/q9f/secp256k1.cr</title>
+  <title>Secp256k1::ECPoint - github.com/q9f/secp256k1.cr</title>
```
Comment (q9f) [inline docs/Secp256k1/EC_Point.html:18]: this file should be removed

File: docs/index.html [hunk 3]
```diff
@@ -40,7 +40,7 @@
   <div class="types-list">
     <ul>
   
-  <li class="parent open" data-id="github.com/q9f/secp256k1.cr/Secp256k1" data-name="secp256k1">
```
Comment (q9f) [inline docs/index.html:43]: can you restore the `open` class here?

File: src/bitcoin.cr [hunk 4]
```diff
@@ -451,7 +446,7 @@ module Secp256k1::Bitcoin
       pk_checksum = hashed_twice[0, 8]
       valid = valid && wif_checksum === pk_checksum
     end
-    return valid
+    valid
```
Comment (q9f) [inline src/bitcoin.cr:449]: i personally prefer having return statements to improve readability but I'm not very opinionated about it, so LGTM

**Review by q9f — APPROVED** [review]
LGTM if you could remove the two obsolete docs

File: docs/index.html (hunk 3 above)
Comment (cserb) [reply docs/index.html:43]: I don't know how to do that, since this is generated by `crystal doc`

File: src/bitcoin.cr (hunk 4 above)
Comment (cserb) [reply src/bitcoin.cr:449]: I actually concur, but the crystal/ruby community decided to use return only for early break outs

**Pull request merged**